### PR TITLE
ox 0.2.5 (new formula)

### PR DESCRIPTION
**Pull request by vladimyr**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

### CODE REVIEW

File: Formula/ox.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  desc "Independent Rust text editor that runs in your terminal"
+  homepage "https://github.com/curlpipe/ox"
+  url "https://github.com/curlpipe/ox/archive/0.2.5.tar.gz"
+  sha256 "873eb447029508bc3fd1d7dda8803d79a7b107a7a903399947f4eac6ae671176"
+  license "GPL-2.0-only"
+  head "https://github.com/curlpipe/ox.git"
+
+  livecheck do
+    url "https://github.com/curlpipe/ox/releases/latest"
+    regex(%r{href=.*?/tag/([^"' >]+)["' >]}i)
```
Comment (chenrui333) [inline Formula/ox.rb:11]:
```suggestion
    regex(%r{href=.*?/tag/v?(\d+(?:\.\d+)+)["' >]}i)
```

**Comment by chenrui333** [comment]
Other than livecheck, lgtm.

File: Formula/ox.rb (hunk 1 above)
Comment (vladimyr) [reply Formula/ox.rb:11]: Yeah, sure, it makes more sense 👍

**Comment by vladimyr** [comment]
@chenrui333 Sorry for going off-topic but I'm obviously out of sync with current formula authoring practices...
Do I actually need to add `livecheck` block here? Guessing from `brew livecheck --debug` output I could simply let it fall back to git (tag) strategy? 🤔

**Comment by chenrui333** [comment]
yeah, not a bad idea to include the livecheck block (in fact, we added lots of similar livecheck blocks recently).

**Comment by chenrui333** [comment]
in terms of new formula authoring, I am not quite on top of that.

**Review by chenrui333 — DISMISSED** [review]

**Comment by SMillerDev** [comment]
The git tag strategy is good, we don't want to maintain a block we don't actually need.

**Comment by vladimyr** [comment]
> The git tag strategy is good, we don't want to maintain a block we don't actually need.

Yup, figured that out by reading livecheck implementation so I removed the block. It was the `fnm` formula's recent addition that sent me in the wrong direction. If I understand things correctly it features a redundant live check block just as you described...
Regardless of that thanks both of you @chenrui333 and @SMillerDev for updating me and bringing me up to speed (as always)! 👍

**Review by SMillerDev — APPROVED** [review]
Thanks @vladimyr  ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Review by chenrui333 — APPROVED** [review]

**Pull request closed without merging**